samples/drivers/spi_flash/src/main.c

```diff
@@ -33,6 +33,7 @@
 void single_sector_test(const struct device *flash_dev)
 {
 	const uint8_t expected[] = { 0x55, 0xaa, 0x66, 0x99 };
+	const uint8_t erased[sizeof(expected)] = { 0xff, 0xff, 0xff, 0xff };
```

Comment:
You need to pick erase value from device drivers. There is at least one device in tree that erases to zero.

Reply:
This is interesting. The sample explicitly states it interacts with SPI NOR serial flash devices, and the sample.yaml filters on SPI NOR flash. Could an SPI NOR flash have erase value other than 0xff?

Reply:
We set the value to flash parameters so we should probably use it here:
https://github.com/zephyrproject-rtos/zephyr/blob/3d29c9fe546cd42554becee9dddcbe2e9e73d5cc/drivers/flash/spi_nor.c#L237-L240

Reply:
Shouldn't this be changed before merging?

Reply:
> Shouldn't this be changed before merging?

@tbr-tt Care to fix this?

Reply:
@tbr-tt Please address the issue mentioned by @henrikbrixandersen before Stale label expires the PR.

Reply:
@tbr-tt Ping